lib/registration.pm

```diff
@@ -366,6 +367,18 @@ sub _yast_scc_addons_handler {
     }
 }
 
+sub skip_package_hub_if_necessary {
+    my ($addon) = @_;
+    if (is_sle('15-SP2+') && $addon eq 'phub') {
+        if (check_var('FLAVOR', 'Online')) {
+            record_soft_failure('bsc#1151373 - Missing or broken repository after registering module PackageHub');
+        } elsif (check_var('FLAVOR', 'Full')) {
+            record_info('Full media: no PHUB', 'Skipping Package Hub, it is not available on offline scenarios - bsc#1157659');
+        }
+        next;
```

Comment:
I'm not sure if that's going to work. We need this `next` statement in the loop, so I would return true when we need to skip it.
```perl
next if skip_package_hub_if_necessary($addon);
```

Reply:
In the two verification runs it works

Reply:
True... Perl is weird, I would not expect `next` to exit loop outside of subroutine. @foursixnine is that ok? =)

Reply:
Anyway, let's merge to unblock testing.